content/standards/UKGemini/index.html.md

```diff
@@ -1,18 +1,19 @@
 ---
-title: UK Gemini
-keywords: Semantics
-identifier: UK-123
-link: https://www.agi.org.uk/agi-groups/standards-committee/uk-gemini
-isRelatedTo: ISO 19139
 name: UK Gemini
-supersededBy: 2.3
-organisation: Geospatial Commission
-validFrom: Jun-18
-status: active
-startDate: 2020-12-01
-dateAdded: 2020-12-16
-dateUpdated: 2020-12-16
-classification: Metadata
+contraction: GEMINI
+specification:
+  name: 'UK-GEMINI Standard and Inspire Implementing Rules'
+  url: https://www.agi.org.uk/gemini/40-gemini/1037-uk-gemini-standard-and-inspire-implementing-rules/
+  published: '2018-06'
+maintainer: Geospatial Commission
```

Comment:
could we use identifiers for organisations here instead of free text?

Reply:
It's possible to use both (007cfc846f8e7466035553729e058c5067411e10) - free text is for simpler requirements/contributions, but if you think we should require it to be always in the data file, I can amend?